From 03d0f33c9dbc71d018e8f6e21b0424efa16319e8 Mon Sep 17 00:00:00 2001
From: Mathieu Rene <mathieu.rene@gmail.com>
Date: Mon, 20 Feb 2023 12:44:26 -0500
Subject: [PATCH] don't vendor openssl

---
 Cargo.toml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Cargo.toml b/Cargo.toml
index f2a63ae..b641599 100644
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -26,7 +26,7 @@ serde_json = "1.0"
 itertools = "0.10"
 prettytable-rs = { version = "0.10", default-features = false, optional = true }
 clap = { version = "3.2", features = ["derive"] }
-openssl = { version = "0.10", features = ["vendored"] }
+openssl = { version = "0.10" }
 chrono = "0.4"
 tracing = "0.1"
 tracing-bunyan-formatter = { version = "0.3", optional = true }
-- 
2.39.1

